data-upload-ds.R

```diff
@@ -56,6 +56,12 @@ conn <- do.call(pool::dbPool, params)
 sites <- dplyr::tbl(conn, dbplyr::in_schema("data", "Site")) %>%
   dplyr::collect()
 
+## get Visit ID's, Global ID's, and last edited date to compare with source data from Survey123
+visitIDs <- dplyr::tbl(conn, dbplyr::in_schema("data", "Visit")) %>% 
+  dplyr::collect() %>% 
+  dplyr::select(ID,GlobalID,Survey123_LastEditedDate) %>% 
```

Comment:
I think you can swap these lines in order to execute the select statement on the database side...doesn't make a big difference at this point but ought to speed things up with a large dataset